logstreams/src/main/java/io/zeebe/logstreams/impl/LogStorageAppender.java

```diff
@@ -181,7 +190,20 @@ public String getName() {
 
   @Override
   protected void onActorStarting() {
+    try {
+      appendIndex = distributedLog.getLastAppendIndex();
```

Comment:
Doesn't this block the current thread? Can we use async operation for this?

Reply:
Yes my first thought was to block the thread to not start before (servicontainer like start) but this makes here in the model no sense. I think i should use runBlockCurrentPhase or something similar. You're right